tools/tests/conftest.py

```diff
@@ -6,10 +6,12 @@
 REGION = "af-south-1"
 TEST_BUCKET_NAME = "test-bucket"
 SQS_QUEUE_NAME = "test-queue"
+SQS_DEADLETTER_QUEUE_NAME = "test-queue-deadletter"
 TEST_DATA_DIR = Path(__file__).absolute().parent / "data"
 REPORT_FILE = "2021-08-17_update.txt.gz"
 FAKE_STAC_FILE = "fake_stac.json"
 FAKE_LANDSAT_8_BULK_FILE = "fake_landsat_8_bulk_file.csv.gz"
+FAKE_LANDSAT_GAP_REPORT = "landsat_gap_report.txt.gz"
```

Comment:
I don't think we should be putting test-specific environment variables in the conftest.py file. It just adds absstraction that doesn't help anything.

Reply:
FAKE_LANDSAT_GAP_REPORT is there as INVENTORY_MANIFEST_FILE, INVENTORY_DATA_FILE, and others which I'm using in the tests through @pytest.fixture for re-use of TEST_DATA_DIR and REPORT_FOLDER.